### PR TITLE
Story 2 - UI screens for managing products with mock api integration.

### PR DESCRIPTION
**Pull request by pavan-nw**
Screenshot 2020-09-28 at 12 40 23 PM
Screenshot 2020-09-28 at 12 39 59 PM
Screenshot 2020-09-28 at 12 39 43 PM

### CODE REVIEW

**Comment by vghatikar** [comment]
@pavan-nw  The screens look good. I have a suggestion, instead of tab format what if we make a provision to add product on the same screen of View Products? We can provide the **Add Product** button, it opens up the dialogue box which takes all the details on the same screen.

**Comment by pavan-nw** [comment]
> @pavan-nw The screens look good. I have a suggestion, instead of tab format what if we make a provision to add product on the same screen of View Products? We can provide the **Add Product** button, it opens up the dialogue box which takes all the details on the same screen.

Ya, I did the same earlier, but usually pop up should be used for non frequent operations. Like edit operation is rare compared to add.

For add, user will try to add multiple products continuously one after another so popup does not look good

File: front-end/src/components/products/ProductForm.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+    const initialProductCode = selectedProduct ? selectedProduct.code : '';
+    const initialProductName = selectedProduct ? selectedProduct.name : '';
+    const initialPackaging = selectedProduct ? selectedProduct.packaging : '';
+
+    const [selectedPackaging, updatePackaging] = useState(initialPackaging);
+    const [productCode, updateProductCode] = useState(initialProductCode);
+    const [productName, updateProductName] = useState(initialProductName);
+
+    const packagings = [
```
Comment (vghatikar) [inline front-end/src/components/products/ProductForm.tsx:31]: The units are always represented in the small letters. There some exceptions where capitals also are used but those exceptions don't fall here.

**Review by vghatikar — CHANGES_REQUESTED** [review]

File: front-end/src/components/products/ProductForm.tsx (hunk 1 above)
Comment (pavan-nw) [reply front-end/src/components/products/ProductForm.tsx:31]: Sure lets define it how to display, may be I would keep like these as of now else my mock apis wont work.

**Review by vghatikar — APPROVED** [review]
Candidate to Approve.

**Pull request merged**